src/components/NavBar.js

```diff
@@ -13,6 +13,9 @@ const NavBar = ({ handleWriteClick, handleJournalClick }) => {
         <Button onClick={handleJournalClick} className='ui button'>
           Read Journal
         </Button>
+        <Button onClick={handleSignOut} className='ui button'>
```

Comment:
This works fine for our case, but be aware that if we expand to more routes we will need to render this based on state.

Reply:
sure, we will have to make it render conditionally if we add another page.